### DISCUSSION TITLE
How to build a subcutaneous dosing compartment in Mobi? Which parameters should be optimized?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1594] [How-Tos]
Hi! 
I am building a model of a subcutaneous drug. I am wondering:
Is it necessary to add a subcutaneous doing compartment in Mobi?
And which parameters are needed to be added or optimized?
If I want to do it, is there any tutorial or code.zip on this website that I can refer to?

**Comment by wilbertdew** [comment]
Hi @Gre-ux, you can start by dosing your compound in the skin interstitial space, which you can do directly in PK-Sim. See how that behaves first, quite often it seems to do the job for subcutaneous dosing.
Hope this helps!
Wilbert

**Reply by Gre-ux** [reply]
Thanks for your reply! I have already tried what you say. However, the predicted Tmax differ a lot from observed data. 
So I am wondering what can do to improve the modelling performance.

**Reply by YangYu-China** [reply]
> Hi @Gre-ux, you can start by dosing your compound in the skin interstitial space, which you can do directly in PK-Sim. See how that behaves first, quite often it seems to do the job for subcutaneous dosing. Hope this helps! Wilbert
How to do my compound in the skin interstitial space in PK-Sim for subcutaneous dosing? Could you show me a screenshot? Thank you very much.

**Reply by wilbertdew** [reply]
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/8969023/bbbd6e07-fa80-4732-8d5a-0f8499d8e0dc)

**Comment by duoduo982** [comment]
Hi, did you get your problem solved? I have the same problem and would like to ask you for advice!
